# Mapping the emergence of mesenchymal diversity in the developing human intestine and organoids 

Kelli Johnson1, Xiangning Dong1, Yu-Hwai Tsai1, Angeline Wu1, Sydney G. Clark2, Rachel Zwick3, Ian Glass4, Katherine D. Walton1, Ophir Klein3, Jason R. Spence1,5-7

1Department of Internal Medicine, Division of Gastroenterology, University of Michigan Medical School, Ann Arbor, MI 48109, USA  
2Department of Cellular and Molecular Biology, University of Michigan Medical School, Ann Arbor, MI 48109, USA  
3Program in Craniofacial Biology and Department of Orofacial Sciences, University of California, San Francisco, San Francisco, California, USA  
4Department of Pediatrics, Genetic Medicine, University of Washington, Seattle, WA 98195, USA  
5Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, MI 48109, USA  
6Department of Biomedical Engineering, University of Michigan and University of Michigan College of Engineering, Ann Arbor, MI, USA  

- 7Author for correspondence: spencejr@umich.edu.

**Keywords: development, mesenchyme, fibroblast, single-cell sequencing, spatial transcriptomics, Xenium, human, intestine**

**Abbreviations: scRNA-seq, FISH, IF, SEC, FFPE, Xenium, spatial transcriptomics**

## Summary
Understanding how diverse mesenchymal populations are organized during human intestinal development is critical for decoding tissue architecture and function yet remains poorly defined. To construct a comprehensive, tissue-scale map of the developing human small intestine, we leveraged single-cell RNA-sequencing data to build a custom Xenium spatial transcriptomics gene panel covering the diversity of cell types in the human intestine. We focused our analysis on the developing mesenchyme populations (also referred to as fibroblasts or stroma) given the lack of spatiotemporal information about these cell populations. We identified 5 broad mesenchymal populations occupying discrete anatomical niches within the lamina propria and submucosa â€“ the subepithelial cells (SEC), lamina propria fibroblast (LPF), submucosal fibroblasts (SMF), smooth muscle cells (SMC), and CXCL13+ fibroblasts. Our data reveal dynamic spatial remodeling of fibroblast niches during development and establish molecular markers to distinguish these populations. We leverage this high-resolution atlas to benchmark pluripotent stem cell-derived human intestinal organoids differentiated using multiple methods to demonstrate how this foundational resource can be used to dissect intestinal stromal signaling centers, with broad implications for modeling development, regeneration, and disease.

